core/templates/dev/head/pages/topics_and_skills_dashboard/topics_list_directive.html

```diff
@@ -38,6 +40,12 @@
           <span> <[topic.canonical_story_count]> </span>
         </a>
       </td>
+      <td>
+        <a ng-href="<[getTopicEditorUrl(topic.id)]>" class="list-summary">
+          <span ng-if="topic.is_published"> Published! </span>
```

Comment:
Don't need exclamation mark :) It's fun the first time, but could get jarring if you have lots of published topics.

Reply:
Ok, done.